content/guides/browser-automation.md

```diff
@@ -77,6 +77,48 @@ public class ExampleSteps {
 }
 ```
 
+```kotlin
+package class.example;
+
+import cucumber.api.Scenario
+import cucumber.api.java8.En
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.WebDriverWait
+
+class ExampleSteps: En {
+
+    lateinit var driver: WebDriver
```

Comment:
Why late init?

Reply:
Hmm, usually I'd autowire it, in which case I use lateinit var. Maybe not needed here.

Reply:
lateinit prevents having to initialize here, and making WebDriver nullable.
Personally, I prefer lateinit var to private var driver: WebDriver? - null

Reply:
But shouldn't the webdriver be injected then? It is never instantiated.